browser/extensions/brave_extension_management.cc

```diff
@@ -57,10 +51,6 @@ BraveExtensionManagement::BraveExtensionManagement(Profile* profile)
 #endif
   }
   // Make IsInstallationExplicitlyAllowed to be true
-#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
-  AccessById(kEthereumRemoteClientExtensionId)->installation_mode =
-      ManagedInstallationMode::kRecommended;
-#endif
   Cleanup(profile);
 }
```

Comment:
~// Make IsInstallationExplicitlyAllowed to be true~

Reply:
Fixed in ae7a3d2210d080a8e4e2dadf4a1ddb864241ea2c